script/SignAndCheck.s.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson} from "script/SignFromJson.s.sol";
+import {JsonTxBuilderBase} from "src/JsonTxBuilderBase.sol";
+import {MultisigBuilder} from "@base-contracts/script/universal/MultisigBuilder.sol";
+import {ChainAssertions, Types, DeployConfig} from "@eth-optimism-bedrock/scripts/ChainAssertions.sol";
+import {IGnosisSafe} from "@eth-optimism-bedrock/scripts/interfaces/IGnosisSafe.sol";
+
+import {IMulticall3} from "forge-std/interfaces/IMulticall3.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {console} from "forge-std/console.sol";
+import {VmSafe} from "forge-std/Vm.sol";
+
+contract SignAndCheck is SignFromJson {
+    DeployConfig public cfg;
+    VmSafe.AccountAccess[] accesses;
+
+    mapping(address => bool) public loggedAccountAccesses;
+    mapping(address => bool) public loggedAccountDiffs;
+    mapping(address => bool) public loggedAccountHasDiffs;
+
+    function setUp() public {
+        string memory path = string.concat(
+            vm.projectRoot(), "/lib/optimism/packages/contracts-bedrock/deploy-config/", "sepolia", ".json"
+        );
+        cfg = new DeployConfig(path);
+        vm.label(0xd81f43eDBCAcb4c29a9bA38a13Ee5d79278270cC, "StorageSetter");
```

Comment:
Suggestion: these contracts will be used for tests/scripts across repos.  We can move them into a standalone repo with a forge Test / Script contract which inherits these constants.  It will help keep track of active implementations as well.

Reply:
Good suggestion, will take this into future consideration as this repo evolves and becomes more stable.